codegen/tests/cardspb/cards.pb.go

```diff
@@ -65,9 +65,7 @@ func init() {
 	proto.RegisterType((*Card)(nil), "cards.v1.Card")
 }
 
-func init() {
```

Comment:
Remove these protobuf changes please. They're not affected by your change. It looks like `go fmt` isn't formatting your output correctly.